Add documentation

UdayakumarHidakal opened the pull request:
Documentations for the RustBoot

nihalpasham commented:
> Documentations for the RustBoot

Just a few changes:
- can you please fix some of the spellings.
- remove the second header for now.

UdayakumarHidakal commented:
> > Documentations for the RustBoot
> 
> Just a few changes:
> 
>     * can you please fix some of the spellings.
> 
>     * remove the second header for now.

Removed the second header and corrected the spellings.

nihalpasham commented:
I have added a separate git workflow for pull requests. Can you update this pull request with an upstream fetch followed by a push?
